docs/install_linux.md

```diff
@@ -101,6 +101,14 @@ Android users can install via Termux:
 pkg install gh
 ```
 
+### Gentoo
+
+Gentoo Linux users can install from the [main portage tree](https://packages.gentoo.org/packages/dev-util/github-cli):
+
+``` bash
+emerge -av github-cli
```

Comment:
Nice, so there isn't anything else that a Gentoo user needs to set up to be able to run this?

What would be an upgrade command when new versions are released?

Reply:
No, there is nothing else to do. Dependencies are pulled in automatically, then the source is compiled and installed.

Users could upgrade the package by itself using `emerge -u github-cli` if they wished, but it's usually unnecessary and rarely done so. Whenever user issues an `emerge -u @world` or `emerge -uND @world` to upgrade their system, this package will be also upgraded if a new version is available.

Reply:
Thanks for the additional context!

Could you still add upgrade instructions, mostly so we have parity with other installation methods? See how install/upgrade instructions are listed side-by-side in other sections.